docs/integration/kubernetes.md

```diff
@@ -192,598 +192,593 @@ Container and Kubernetes metrics are collected along with labels and tags, which
+|etcd.disk_backend_commit.p90latency|float64| p90 latency of commit operations  |
+|etcd.disk_backend_commit.p95latency|float64| p95 latency of commit operations  |
+|etcd.disk_backend_commit.p99latency|float64| p99 latency of commit operations  |
+|etcd.disk_backend_commit.total_num|uint64| number of commit operations for etcd's disk bakend commits |
+|etcd.disk_backend_commit.duration|float64| duration of commit operations for etcd's disk backend in seconds |
 
 #### WAL
-|Metric Name|Unit|Description|
-|----|----|-----------|
-|etcd.disk_wal_fsync.p50latency|float64||
-|etcd.disk_wal_fsync.p75latency|float64||
-|etcd.disk_wal_fsync.p90latency|float64||
-|etcd.disk_wal_fsync.p95latency|float64||
-|etcd.disk_wal_fsync.p99latency|float64||
-|etcd.disk_wal_fsync.total_num|uint64||
-|etcd.disk_wal_fsync.duration|float64|The latency distributions of fsync called by WAL|
+|Metric Name|Unit| Description                      |
+|----|----|----------------------------------|
+|etcd.disk_wal_fsync.p50latency|float64| p50 latency of fsync operations  |
+|etcd.disk_wal_fsync.p75latency|float64| p75 latency of fsync operations  |
+|etcd.disk_wal_fsync.p90latency|float64| p90 latency of fsync operations  |
+|etcd.disk_wal_fsync.p95latency|float64| p95 latency of fsync operations  |
+|etcd.disk_wal_fsync.p99latency|float64| p99 latency of fsync operations  |
+|etcd.disk_wal_fsync.total_num|uint64| number of fsync (flush to disk) operations for etcd's write-ahead log (WAL) on the disk backend |
+|etcd.disk_wal_fsync.duration|float64| duration of fsync (flush to disk) operations for etcd's write-ahead log (WAL) on the disk backend in seconds |
 
 #### Backend Snapshots
-|Metric Name|Unit|Description|
-|----|----|-----------|
-|etcd.disk_backend_snapshot.p50latency|float64||
-|etcd.disk_backend_snapshot.p75latency|float64||
-|etcd.disk_backend_snapshot.p90latency|float64||
-|etcd.disk_backend_snapshot.p95latency|float64||
-|etcd.disk_backend_snapshot.p99latency|float64||
-|etcd.disk_backend_snapshot.total_num|uint64||
-|etcd.disk_backend_snapshot.duration|float64|The latency distribution of backend snapshots|
+|Metric Name|Unit| Description                                                      |
+|----|----|------------------------------------------------------------------|
+|etcd.disk_backend_snapshot.p50latency|float64| p50 latency of snapshot creation                                 |
+|etcd.disk_backend_snapshot.p75latency|float64| p75 latency of snapshot creation                                 |
+|etcd.disk_backend_snapshot.p90latency|float64| p90 latency of snapshot creation                                 |
+|etcd.disk_backend_snapshot.p95latency|float64| p95 latency of snapshot creation                                 |
+|etcd.disk_backend_snapshot.p99latency|float64| p99 latency of snapshot creation                                 |
+|etcd.disk_backend_snapshot.total_num|uint64| number of snapshot creation for etcd's disk backend              |
```

Comment:
disk backend...

Reply:
I can change if you disagree with my response from https://github.com/sematext/docs/pull/514#discussion_r1293183122